server/flaskr/utils/input_validation.py

```
@@ -21,53 +32,95 @@ def verify_missing_inputs(data: dict, validators: list[InputValidator]) -> list[
+            if value in self.enum.__members__
+            else InputError.EnumMemberNotFound
+        )
+
+
+class ListValidator(InputValidator):
+    def __init__(self, field_name, can_be_empty=True):
+        super().__init__(field_name)
+        self.can_be_empty = can_be_empty
+
+    def validate(self, data) -> InputError:
+        l = data.get(self.field_name, [])
```

Ruff (0.14.14)

[error] Ambiguous variable name: `l`

(E741)